### PR TITLE
fix(icon): fix icon color (VIV-1101)

### PR DESCRIPTION
**Pull request by RichardHelm**

### CODE REVIEW

File: libs/components/src/lib/nav-item/README.md [hunk 1]
```diff
@@ -71,3 +45,25 @@ Within a set of pagination links, set a nav item `aria-current` value to *page*
...
+```html preview
+<vwc-nav>
+	<vwc-nav-item href="#" icon="profile" text="Account"></vwc-nav-item>
+	<vwc-nav-item href="#" icon="profile" text="Account" aria-current="page"></vwc-nav-item>
+</vwc-nav>
+<h4>Icon only</h4>
+<vwc-nav>
+	<vwc-nav-item href="#" icon="profile" aria-label="Account"></vwc-nav-item>
+	<vwc-nav-item href="#" icon="profile" aria-label="Account" aria-current="page"></vwc-nav-item>
+</vwc-nav>
```
Comment (olaf-k) [inline libs/components/src/lib/nav-item/README.md:68]:
```suggestion
<vwc-nav>
	<vwc-nav-item href="#" icon="profile" text="Account"></vwc-nav-item>
	<vwc-nav-item href="#" icon="profile" aria-label="Account"></vwc-nav-item>
</vwc-nav>
```
I understand the rationale to merge the two examples as they refer to the same attribute but we should keep them as short and to the point as possible. (the usage of aria-current is explained right before, there's no need to duplicate it here.)

**Review by olaf-k — CHANGES_REQUESTED** [review]

**Review by olaf-k — APPROVED** [review]

**Review by rachelbt — CHANGES_REQUESTED** [review]
this is not a fix - the icon need to be in that color - @AyalaBu please

**Comment by AyalaBu** [comment]
In the nav-item the icons is supposed to have the same color as text,
since it's also clickable.

**Comment by RichardHelm** [comment]
@rachelbt I've followed the Figma design where the icon has the same color as the text

**Review by rachelbt — APPROVED** [review]

**Pull request merged**